binary_wheel_builder/cli/main.py

```diff
@@ -25,6 +25,11 @@ def _parse_args(args) -> Namespace:
         type=int,
         help="Number of parallel workers to use at most for building wheels"
     )
+    parser.add_argument(
+        '--version',
+        action='version',
+        version='Wheel Builder 1.0.0'
```

Comment:
chore: Please remove this hardcoded version number here (and potentially the entire version parameter)

Currently the version of the application is not stored to any file or available at runtime apart from using `importlib` for this, which tends to be rather unstable.

I would omit this one from the PR and make that change later on, with a proper version being available at runtime.

Reply:
Ok, maybe it´s not the right moment to add this argument, I will remove it, I was thinking about it and is not necessary at all, but thanks for talk me about that.

Reply:
But good idea 👍 - will definitely add this